governance/votes/Executive Vote - April 30, 2025.md

```diff
@@ -0,0 +1,237 @@
+
+If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Sky Protocol:
+- STAR2 Allocation System will be updated as described below.
+- The [GSM Pause Delay](https://sky-atlas.powerhouse.io/#A.1.8.2.1_Pause_Delay-a98b8227-95f6-4711-9d8d-f52cbc6ad2d0|0db30758e055) will be increased.
+- EMSP_SPBEAM_HALT Standby Spell at [0xf0C6e6Ec8B367cC483A411e595D3Ba0a816d37D0](https://etherscan.io/address/0xf0C6e6Ec8B367cC483A411e595D3Ba0a816d37D0) will be added to the Chainlog.
+- **3 million USDS** will be transferred to top off the Integration Boost at [0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7](https://etherscan.io/address/0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7).
+- **5 million USDS** and **30 million SKY** will be transferred to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+- A Spark proxy spell at [0x9362B8a15ab78257b11a55F7CC272F4C4676C2fe](https://etherscan.io/address/0x9362B8a15ab78257b11a55F7CC272F4C4676C2fe) will be executed.
+- A STAR2 proxy spell at [0x0c9CC5D5fF3baf096d29676039BD6fB94586111A](https://etherscan.io/address/0x0c9CC5D5fF3baf096d29676039BD6fB94586111A) will be executed.
+
+
+**Voting for this executive proposal will place your MKR/SKY in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://sky-atlas.powerhouse.io/#A.1.8.2.1_Pause_Delay-a98b8227-95f6-4711-9d8d-f52cbc6ad2d0|0db30758e055). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Sky Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to [**18 hours**](https://sky-atlas.powerhouse.io/#A.1.8.2.1.2_Pause_Delay_Current_Value-09d2514b-3169-4755-a654-2c774456980d|0db30758e055d2d0).
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Sky Protocol.
+
+---
+
+## Proposal Details
+
+### STAR2 Allocation System Updates
+
+- **Authorization**: Governance [Poll 1235](https://vote.makerdao.com/polling/QmepaQjk), [Poll 1243](https://vote.makerdao.com/polling/QmedB3hH)  
+- **Proposal**: [Forum Post](https://forum.sky.money/t/technical-test-of-the-star2-allocation-system/26289)
+
+If this executive proposal passes, then the following updates to the STAR2 Allocation System will be made.
+
+#### Whitelist STAR2 ALMProxy
+
+- The STAR2 ALMProxy will be whitelisted on the LitePSM by performing a `kiss` call.
```

Comment:
Is `MCD_LITE_PSM_USDC_A.kiss(0x491EDFB0B8b608044e227225C715981a30F3A44E)` "example code" as per the poll or is it actual code? If it is, we should probably add it here as we usually do.